### ISSUE TITLE
Empty netMHC4.0 fasta files for GRCh37

### ISSUE THREAD

**Issue by bbhinder** [ChrisMaherLab/INTEGRATE-Neo#5]
Hi,

I was able to successfully install and test run INTEGRATE-Neo on example2 (using GRCh38).

My issue is that when I try to run this on my dataset (which is processed using GRCh37), I get empty fasta files (e.g.,netMHC4.0.k10.fasta) in the fusion_antigen_out/tmp folder. Your advise in the matter will be very helpful.

I downloaded the GRCh37 files from the links provided in your README file i/e., ftp://ftp.ensembl.org/pub/release-75/fasta/homo_sapiens/dna/Homo_sapiens.GRCh37.75.dna_sm.primary_assembly.fa
Also downloaded the gtf from your link and converted it into model as follows:
./gtfToGenePred -genePredExt -geneNameAsName2 Homo_sapiens.GRCh37.75.gtf Homo_sapiens.GRCh37.75.genePred

I had converted my result file (STARfusion) into bedpe file with the same 11-column format as provided in your examples fusions.bedpe file.

Thanks you for your help!

**Comment by jin-44** [comment]
I think I got the same issue. Did you fix it?

**Comment by pdorsaint** [comment]
Hi, how did you convert your STARfusion result file into bedpe format?

**Comment by mjinkm** [comment]
Yes, I believe pdorsaint has asked the critical question.  INTEGRATE-Neo performs correctly when run with fusion BEDPE files generated from GRCh37-aligned BAMs via INTEGRATE-fusion, so it seems likely the error in this case is in the conversion of your STARfusion results to BEDPE.

**Comment by GAOwy123** [comment]
I got STAR-Fusion results and I change the format to BEDPE. The fusion.bedpe.subset  is empty. I think this is because my fusion.BEDPE is not fit for this INTEGRATE-Neo.

**Comment by GAOwy123** [comment]
This is the example fusions.bedpe,  the start or end position is -1 ?
12	-1	14456634	21	-1	41489593	ATF7IP>>TMPRSS2	3	+	-	1
12	-1	14457295	21	-1	41489593	ATF7IP>>TMPRSS2	3	+	-	42
21	41497936	-1	12	14460493	-1	TMPRSS2>>ATF7IP	3	-	+	16
